app/shared/profile-commands.js

```diff
@@ -14,6 +14,8 @@
 
 import * as profileCommandTypes from './constants/profile-command-types';
 
+import { ReadabilityResult } from '../shared/types';
```

Comment:
Needing to actually require() code for flow is kind of gross.  I guess dead code elimination could prune that out in a packaged build though.